index.html

```diff
@@ -203,10 +203,10 @@ <h3 class="tech-title" id="skills-title">Skills</h3>
         <h4 id="form-message">I'm always interested in hearing about 
           new projects, 
           so if you'd like to chat please get in touch.</h4>
-        <form id="contact-form">
-          <input type="text" id="name" name="name" placeholder="Full name">
-          <input type="email" id="email" name="email" placeholder="Email address">
-          <input type="text" id="message" name="message" placeholder="Hey I’d like to connect to...">
+        <form action="https://formspree.io/f/mqkovdqr" method="POST" id="contact-form">
+          <input type="text" id="name" name="name" placeholder="Full name" maxlength="30" required>
+          <input type="email" id="email" name="email" placeholder="Email address" required>
+          <textarea type="text" id="message" name="message" placeholder="Hey I’d like to connect to..." maxlength="500" required></textarea>
```

Comment:
[OPTIONAL]
- I kindly suggest you remove the type attribute from the texarea properties, it is not an input element and as such it doesn't require a type :+1: :sparkles:

Reply:
@Efiamotu-1  thanks for your time. Best wishes for you.